index.html

```diff
@@ -1,16 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Cake webpage</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
 
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
+  <body>
+    <main class="homepage-main">
+      <section class="homepage-hero">
+        <div class="one-one">
```

Comment:
I think these are leftovers from our coding mob

Reply:
what's their scope? I deleted them in DOM and could see no relevant changes

Reply:
I used .one-one class to create hamburger sign in mobile version
<img width="404" alt="Screenshot 2022-11-10 at 15 29 44" src="https://user-images.githubusercontent.com/12444326/201137134-f5e215f6-faa2-4327-b3b3-ba4fff448209.png">

Reply:
got it

Reply:
yes, you was right.
there were menu in tablet mode, I returned it :)